### PR TITLE
831665: GettingStarted UG for SfAccordion.

### PR DESCRIPTION
**Pull request by RasikaPalanisamy**
Created GettingStarted Ug for MAUI SfAccordion.

### CODE REVIEW

File: MAUI/Accordion/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,790 @@
...
+                            AnimationDuration="150" /> 
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1" %}
+    accordion.AnimationDuration = 150;
+{% endhighlight %}
+{% endtabs %}
+
+## Animation easing
+
+The `SfAccordion` allows you to customize the rate of change of a parameter over time or the animation style of an accordion item by using the `AnimationEasing` property. By default, the animation easing is set to `Linear`.
```
Comment (M-A32) [inline MAUI/Accordion/getting-started.md:738]: The `SfAccordion` allows you to customize the rate of change of a parameter over time or the animation style of an accordion item by using the `AnimationEasing` property. By default, the animation easing is set to `Linear.`

File: MAUI/Accordion/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,790 @@
...
+                                AnimationEasing="SinOut" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1" %}
+    accordion.AnimationEasing = Syncfusion.Maui.Expander.AnimationEasing.SinOut;
+{% endhighlight %}
+{% endtabs %}
+
+## Auto scroll position
+
+The `SfAccordion` allows you to customize the scroll position of the expanded accordion item using the `AutoScrollPosition` property.  By default, the auto scroll position is set to `MakeVisible`.
```
Comment (M-A32) [inline MAUI/Accordion/getting-started.md:752]: The `SfAccordion` allows you to customize the scroll position of the expanded accordion item using the `AutoScrollPosition` property.  By default, the auto-scroll position is set to `MakeVisible.`

File: MAUI/Accordion/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,790 @@
...
+                             AutoScrollPosition="Top"/>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1" %}
+    accordion.AutoScrollPosition = Syncfusion.Maui.Accordion.AutoScrollPosition.Top;
+{% endhighlight %}
+{% endtabs %}
+
+## Expand mode
+
+The `SfAccordion` allows you to expand single or multiple items by using the `ExpandMode` property. By default, the expand mode is set to `Single`.
```
Comment (M-A32) [inline MAUI/Accordion/getting-started.md:766]: The `SfAccordion` allows you to expand single or multiple items using the `ExpandMode` property. By default, the expanded mode is set to `Single.`

**Review by M-A32 — CHANGES_REQUESTED** [review]

File: MAUI/Accordion/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,790 @@
...
+                            ExpandMode="Multiple" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1" %}
+    accordion.ExpandMode = Syncfusion.Maui.Accordion.ExpandMode.Multiple;
+{% endhighlight %}
+{% endtabs %}
+
+## Item spacing
+
+The `SfAccordion` allows you to customize the vertical spacing between the accordion items by using the `ItemSpacing` property. The default value is `6.0d`.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/Accordion/getting-started.md:780]: Addressed the concerns provided.

File: MAUI/Accordion/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,790 @@
...
+
+Create a new blank (.NET Standard) application in Visual Studio for Maui. 
+
+### Adding accordion in Maui: 
+
+To add the accordion to your application, follow these steps: 
+
+1. Add required assemblies. 
+2. Import the control namespace as `xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Accordion;assembly=Syncfusion.Maui.Expander` in XAML Page. 
+3. Create an instance of accordion control and add as content.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/Accordion/getting-started.md]: Addressed the concerns provided.

File: MAUI/Accordion/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,790 @@
...
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Defining accordion items 
+
+`SfAccordion` is a layout control with vertically stacked list of accordion `Items` that comprise of `Header` and `Content`. You can load any View in `Header` and `Content`. Users can expand or collapse the Content view by tapping `Header`.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/Accordion/getting-started.md]: Addressed the concerns provided.

File: MAUI/Accordion/getting-started.md [hunk 7]
```diff
@@ -0,0 +1,790 @@
...
+
+Now, run the application to render the following output.
+
+![Defining accordion items](Images/getting-started//maui-accordion-with-defining-accordion-items.png)
+
+N> When adding the template control inside the `StackLayout` or `Grid` with a height set to `Auto`, the child element will not receive the height changes at runtime. Since the `SfAccordion` is a template-based control, the default height value cannot be determined. Therefore, it is recommended to provide the `HorizontalOptions` and `VerticalOptions` as `FillAndExpand` options for the control.
+
+## Animation duration
+
+The `SfAccordion`allows you to customize the expanding and collapsing of accordion items by using the `AnimationDuration` property. By default, the animation duration is set to `250 milliseconds`.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/Accordion/getting-started.md]: Addressed the concerns provided.

File: MAUI/Accordion/getting-started.md (hunk 1 above)
Comment (RasikaPalanisamy) [reply MAUI/Accordion/getting-started.md:738]: Addressed the concerns provided.

File: MAUI/Accordion/getting-started.md [hunk 8]
```diff
@@ -0,0 +1,790 @@
...
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Defining accordion items 
+
+`SfAccordion` is a layout control with vertically stacked list of accordion `Items` that comprise of `Header` and `Content`. You can load any View in `Header` and `Content`. Users can expand or collapse the Content view by tapping `Header`. 
+
+Here, Grid is loaded in Header and Content of accordion items.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/Accordion/getting-started.md]: Addressed the concerns provided.

File: MAUI/Accordion/getting-started.md [hunk 9]
```diff
@@ -0,0 +1,790 @@
+---
+layout: post
+title: Getting started | SfAccordion | MAUI | Syncfusion
+description: This section described about getting started with MAUI Accordion (SfAccordion) control and more details.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/Accordion/getting-started.md]: Here, description should be in small.

**Review by PiruthivirajM — APPROVED** [review]
Fine

**Pull request merged**